### PR TITLE
Add tests for validate.py

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@sarina this is in preparation for https://openedx.atlassian.net/projects/LOC/issues/LOC-93

File: tests/data/validation_problems.po [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+msgid_plural "1. There are {num} things"
+msgstr[0] "1. Estas {num} objectos"
+msgstr[1] "1. Estas {nomx} objectos"
+
+msgid "2. There are {num} things"
+msgid_plural "2. There are {num} things"
+msgstr[0] "2. Estas {nomx} objectos"
+msgstr[1] "2. Estas {num} objectos"
+
+# TODO: This doesn't raise a validation error: we check that the combination of
```
Comment (sarina) [inline tests/data/validation_problems.po]: where are the jira tickets representing these TODOs?

File: tests/test_validate.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+HERE = Path(__file__).dirname()
+TEST_DATA = HERE / "data"
+
+
+class TestValidate(TestCase):
+    """
+    Tests functionality in i18n/validate.py, because if Sarina weren't
+    quitting, she'd be making me write tests.
```
Comment (sarina) [inline tests/test_validate.py]: wat

Comment (sarina) [inline tests/test_validate.py]: :astonished: :scream:

**Comment by sarina** [comment]
OK, these seem reasonable to me, :+1:

**Pull request merged**